tests/ci/spec_to_test_mapping.py

```diff
@@ -42,6 +42,10 @@
         'scotch',
         'zlib-devel'
     ],
+    '../components/parallel-libs/fftw/SPECS/fftw.spec': [
```

Comment:
The leading `../` will be removed in the next commit!
It seems this typo didn't break the test execution ...

Reply:
Answering to myself: If the key is misspelled then `setup_slurm_and_run_tests.sh` cannot find it and it does not execute any tests.